pkg/oc/cli/admin/prune/imageprune/testutil/util.go

```diff
@@ -81,23 +81,23 @@ func ImageWithLayers(id, ref string, configName *string, layers ...string) image
 	}
 
 	if configName != nil {
-		image.DockerImageMetadata = imageapi.DockerImage{
-			ID: *configName,
-		}
+		// image.DockerImageMetadata = imagev1.DockerImage{
+		// 	ID: *configName,
+		// }
```

Comment:
Is this a TODO? Do we still need to find a way to modify the contents of imagev1.DockerImage metadata?

Reply:
Yeah, that was that failing unit tests that I fixed already.